lxd/network/driver_bridge.go

```diff
@@ -2622,6 +2682,30 @@ func getTunnels(config map[string]string) []string {
 	return tunnels
 }
 
+// getFans populates slice with bridges with bridge.mode == "fan".
+func getFans(config map[string]string, name string) []string {
```

Comment:
This function seems a bit odd to me.

Within the context of a single network's `config` map, there can be only one `bridge.mode` config key.

https://documentation.ubuntu.com/lxd/latest/reference/network_bridge/#network-bridge-network-conf:bridge.mode

So why do we need to iterate through the config in order to detect if its set?

It also isn't considering `bridge.mode == "fan"` which the doc block comment says it should.

Reply:
Indeed, thanks for calling it out, I suspect this is a case of me extrapolating getTunnels() to the fan use case without really thinking it through in the heat of the moment.